File: Fibonacci.java
import java.util.*;
class Fibonacci
{
    public static void series ()
    {
        System.out.println("Enter a number");
        Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();
        int a=0;int b=1;int c=0;
        System.out.println("Fibonacci Series");
        System.out.println(a+" "+b);
        while(c<n)
        {
        c=a+b;
        System.out.println(" "+ c);
        a=b;
        b=c;
    }
    }
}

File: Fibonacci.py
t1, t2, t3 = 0, 1, 0
n = int(input("Enter the range: "))
print("The Fibonacci Series is:")
print(t1)
for i in range(n-1):
    t1 = t2
    t2 = t3
    t3 = t1 + t2
    print(t3)